gulpfile.babel.ts

```diff
@@ -113,6 +109,13 @@ async function getVersion(): Promise<string> {
   const commit = results[0].toString().trim().slice(0, 16);
   const isModified = /^\s*M/m.test(results[1].toString());
 
+  const packageJson = JSON.parse(
+    await fs.promises.readFile(
+      path.join(__dirname, 'packages/core/package.json'),
+      'utf8'
+    )
+  );
+
   let version = `${packageJson.version}-${Date.now()}-${commit}`;
```

Comment:
An aside: we could use this version string to determine if the build was an npm build or remote. This would give us insight into how many SDK users are still using MV2.

Something like

```suggestion
  let version = `${packageJson.version}-${Date.now()}-${commit}`;

  if (args.remote) {
    version += '-REMOTE'
  }
```